recipe/meta.yaml

```diff
@@ -7,40 +7,29 @@ package:
 source:
   fn: openblas-{{ version }}.tar.gz
   url: https://github.com/xianyi/OpenBLAS/archive/v{{ version }}.tar.gz
-  sha256: 5ef38b15d9c652985774869efd548b8e3e972e1e99475c673b25537ed7bcf394
-  patches:
-    - gh_1247.patch
-    - gh_1259.patch
-    - gh_1256.patch
-    - gh_1264.patch
-    - gh_1279.patch
-    - gh_1323.patch
-    - gh_1350.patch
-    - gh_1377.patch
-    - gh_1417.patch
+  sha256: cf51543709abe364d8ecfb5c09a2b533d2b725ea1a66f203509b21a8e9d8f1a1
 
 build:
-  number: 8
-  skip: true  # [win32]
-  features:
-    - vc14    # [win]
-  msvc_compiler: 14.0
+  number: 0
+  skip: true  # [win and vc!=14]
 
 requirements:
   build:
-    - toolchain
-    - perl 5.22.2.1
-    - gcc              # [unix]
-    - vc 14            # [win]
-    - flang            # [win]
-    - cmake            # [win]
-    - curl             # [win]
-    - jom              # [win]
+    - {{ compiler("c") }}
+    - {{ compiler("fortran") }}  # [unix]
+    - perl
+    - flang                      # [win]
+    - cmake                      # [win]
+    - curl                       # [win]
+    - jom                        # [win]
+
+  host:
+    - libflang                   # [win]
+    - libgfortran-ng >=3.0       # [osx]
+    - libgfortran-ng >=4.9       # [linux]
```

Comment:
Can you remove the two lines above?

Reply:
It failed to build without these being present.

Reply:
Do you have a log?

Reply:
Here's one.

ref: https://circleci.com/gh/conda-forge/openblas-feedstock/130